base/dcp_client_test.go

```diff
@@ -311,13 +311,117 @@ func TestDCPClientMultiFeedConsistency(t *testing.T) {
 				// only vbucket 0 should have rolled back, expect mutation count to be only vbucketZero
 				mutationCount := atomic.LoadUint64(&mutationCount)
 				require.Equal(t, int(vbucketZeroExpected), int(mutationCount))
+				// check the rolled back vBucket has in fact closed the stream after its finished
+				numVBuckets := len(dcpClient.activeVbuckets)
+				require.Equal(t, uint16(0), uint16(numVBuckets))
 			case <-feed3Timeout:
 				t.Errorf("timeout waiting for first one-shot feed to complete")
 			}
 		})
 	}
 }
 
+func TestContinuousDCPRollback(t *testing.T) {
+	SetUpTestLogging(t, LevelDebug, KeyDCP)
+
+	if UnitTestUrlIsWalrus() {
+		t.Skip("This test only works against Couchbase Server")
```

Comment:
```suggestion
		t.Skip("This test requires DCP feed from gocb and therefore Couchbase Sever")
```

If we ever implement DCP in walrus.